src/data_structures.py

```diff
@@ -19,8 +20,19 @@ class SlackMessage:
     replies: list[SlackThreadMessage]
 
 
+class ChannelType(Enum):
```

Comment:
If you already using python 3.11 it's could be a  good idea to use strEnum here.